src/OpenTK.Windowing.Common/EventingModels/KeyboardCharModEventArgs.cs

```diff
@@ -0,0 +1,59 @@
+//
+// KeyboardCharModEventArgs.cs
+//
+// Copyright (C) 2019 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+using System;
+using OpenToolkit.Windowing.Common;
+using OpenToolkit.Windowing.Common.Input;
+
+namespace OpenToolkit.Windowing.EventingModels
```

Comment:
Maybe we should refactor this to a different namespace. Either OpenToolkit.Windowing.Events or just OpenToolkit.Windowing imo.

Reply:
I agree that OpenToolkit.Windowing would be for the best.

Reply:
Fixed in latest commit.